src/mono/wasm/runtime/rollup.config.js

```diff
@@ -159,4 +172,47 @@ function checkFileExists(file) {
     return fs.promises.access(file, fs.constants.F_OK)
         .then(() => true)
         .catch(() => false);
-}
\ No newline at end of file
+}
+
+function regexReplace(replacements = []) {
+    const filter = createFilter("**/*.ts");
+
+    return {
+        name: "replace",
+
+        renderChunk(code, chunk) {
+            const id = chunk.fileName;
+            if (!filter(id)) return null;
+            return executeReplacement(this, code, id);
+        },
+
+        transform(code, id) {
+            if (!filter(id)) return null;
+            return executeReplacement(this, code, id);
+        }
+    };
+
+    function executeReplacement(self, code, id) {
+        // TODO use MagicString for sourcemap support
```

Comment:
is this going to break all our source maps? or just for the lines/functions that got changed

Reply:
We don't have source map anyway, because the way how emscripten concatenates multiple files.
If we want to fix it in the future, we would have pipe thru typescript->rollup->emscripten-closure compilers.